courses_api/filters.py

```diff
@@ -8,6 +8,25 @@
 
 class CourseFilter(filters.BaseFilterBackend):
 
+    def order_by(self, request):
+        """Get the "sort" parameter from the request
+
+        Returns:
+            tuple: value that can be fed to a .order_by() directive.
+        """
+        sort_param = request.QUERY_PARAMS.get("sort")
+        if sort_param == "title":
+            return ("title",)
+        elif sort_param == "enrollment_start_date":
+            return ("enrollment_start_date",)
+        elif sort_param == "-enrollment_start_date":
+            return ("-enrollment_start_date",)
+        elif sort_param == "start_date":
+            return ("start_date",)
+        elif sort_param == "-start_date":
+            return ("-start_date",)
+        return ("-score",)
```

Comment:
je ne comprends pas trop le but du `if else`. Enfin, il m'a fallu un peu de temps pour comprendre.
J'imagine que c'est pour tester que les sort params sont bien définis et ne valent pas n'importe quoi.

Pourquoi ne pas faire un truc dans le genre : `if sort_param in legal_sorts : return (sort_param,)`
Avec legal_sort : `set("title", "enrolment...")